From 46bf01e2c8aa4341944aeba4df5e63c7c34d771f Mon Sep 17 00:00:00 2001
From: MohammadHossein Qanbari <mohammad.qanbari@qt.io>
Date: Thu, 18 Sep 2025 17:01:36 +0200
Subject: [PATCH 168/553] CRA review: src/widgets/kernel/qsizepolicy.*

The security level for  qsizepolicy.* files are tagged as significant.

QUIP: 23
Fixes: QTBUG-140350
Task-number: QTBUG-135741
Pick-to: 6.8
Change-Id: I858ea681cd3d802a7783870a59148fc4fee0b1c2
Reviewed-by: Volker Hilsheimer <volker.hilsheimer@qt.io>
(cherry picked from commit b2514cf4520552912b213fc757faa3e2fcab1d6a)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/widgets/kernel/qsizepolicy.cpp | 1 +
 src/widgets/kernel/qsizepolicy.h   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/widgets/kernel/qsizepolicy.cpp b/src/widgets/kernel/qsizepolicy.cpp
index e93e3bc89ad..9f5e00df370 100644
--- a/src/widgets/kernel/qsizepolicy.cpp
+++ b/src/widgets/kernel/qsizepolicy.cpp
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #include "qsizepolicy.h"
 
diff --git a/src/widgets/kernel/qsizepolicy.h b/src/widgets/kernel/qsizepolicy.h
index af964bf6aff..393b97e5ca2 100644
--- a/src/widgets/kernel/qsizepolicy.h
+++ b/src/widgets/kernel/qsizepolicy.h
@@ -1,5 +1,6 @@
 // Copyright (C) 2016 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
+// Qt-Security score:significant reason:default
 
 #ifndef QSIZEPOLICY_H
 #define QSIZEPOLICY_H
-- 
2.51.2

